store/store.go

```diff
@@ -10,7 +12,15 @@ import (
 )
 
 func NewCommitMultiStore(db dbm.DB) types.CommitMultiStore {
-	return rootmulti.NewStore(db, log.NewNopLogger())
+	// Developers may enable rootmulti logging by setting env LOG_ROOTMULTI=1
+	if len(os.Getenv("LOG_ROOTMULTI")) > 0 {
```

Comment:
Which of these makes the most sense?

* passing the app the NewCommitMultiStore
* passing the logging level desired to NewCommitMultiStore
* configuring the logger in the baseapp and supplying it to NewCommitMultiStore

Also not sure where such a setting would go in app.toml (other than root?) since store spans quite a number of functions/reactors.

Reply:
Removed this change to eliminate complexity within this PR